Vagrantfile

```diff
@@ -13,7 +13,7 @@ Vagrant.configure(2) do |config|
   # the path on the host to the actual folder. The second argument is
   # the path on the guest to mount the folder. And the optional third
   # argument is a set of non-required options.
-  # config.vm.synced_folder "../data", "/vagrant_data"
+  config.vm.synced_folder "../FundraisingDonations", "/fundraising-donations"
```

Comment:
Except for actual package management (composer file), I think we should leave it to the individual dev to sync the context into the main project - it's bound to break if it references locations on disk